### DISCUSSION TITLE
Unable to fetch some playlists

### DISCUSSION THREAD

**Discussion by kuthq** [sigma67/ytmusicapi#408] [General]
Hey, I have trouble getting any data (get_playlist(playlist_id)) for example this playlist:

https://music.youtube.com/playlist?list=RDCLAK5uy_n4JlAuNLjhr4Sz8BQVS9IfeQiiKnx2_bc

While this works:

https://music.youtube.com/playlist?list=RDCLAK5uy_l6lSnRziP_Qt6EA5-nmhXZ3uQz76JRZw4

**Comment by kuthq** [comment]
Maybe it has to do something with the location from where the request comes because it is a German Playlist?

**Reply by sigma67** [reply]
If you can get it on the web you should be able to get it with ytmusicapi. What error are you getting?

**Reply by kuthq** [reply]
In the German Wifi I get this: 
ERROR:root:Playlist RDCLAK5uy_lx_HcGQ3dqhBbBk3aaZPWoy2trdcdhfio error: 'playlistId'
for this playlist: https://music.youtube.com/playlist?list=RDCLAK5uy_lx_HcGQ3dqhBbBk3aaZPWoy2trdcdhfio

This one works:
(the german version of "RELEASED") 
https://music.youtube.com/playlist?list=RDCLAK5uy_n4JlAuNLjhr4Sz8BQVS9IfeQiiKnx2_bc 

Ps: fetching the German RELEASED Playlist worked until 10th July 11pm

**Reply by kuthq** [reply]
<img width="1440" alt="Bildschirmfoto 2023-07-13 um 13 51 12" src="https://github.com/sigma67/ytmusicapi/assets/112192000/5763d8a0-59bf-45ab-a7d5-9f7d1c767583">
<img width="1440" alt="Bildschirmfoto 2023-07-13 um 13 51 06" src="https://github.com/sigma67/ytmusicapi/assets/112192000/2096cf34-bee1-4d5c-94c2-28468d4ac191">


For the not fetchable playlist the is also the cover missing etc. Are you crawling the html? Maybe it breaks the code without these elements

**Reply by kuthq** [reply]
<img width="1125" alt="Bildschirmfoto 2023-07-14 um 14 59 39" src="https://github.com/sigma67/ytmusicapi/assets/112192000/2be94d83-3fdc-45a0-bc65-29278646e917">

for the not fetchable playlist there is no playlistId in this area.

And there is no 'header' element in the response.

<img width="1153" alt="Bildschirmfoto 2023-07-14 um 15 25 36" src="https://github.com/sigma67/ytmusicapi/assets/112192000/453b2e46-e58d-4440-adab-f7768a1442d7">



Are you able to fetch both RELEASED playlists with the api?

**Reply by kuthq** [reply]
@sigma67 I tried the location parameter, but I did not change the response. Maybe using a proxy is the only way
